packages/tokens-studio-for-figma/src/plugin/createLocalVariablesInPlugin.ts

```diff
@@ -24,9 +24,21 @@ export default async function createLocalVariablesInPlugin(tokens: Record<string
   if (!checkSetting) {
     themeInfo.themes.forEach((theme) => {
       if (!selectedThemes || (selectedThemes && selectedThemes.includes(theme.id))) {
-        const collection = figma.variables.getLocalVariableCollections().find((vr) => vr.name === (theme.group ?? theme.name));
+        let collection;
+        if (figma.variables.getLocalVariableCollections().find((vr) => vr.id === theme.$figmaCollectionId)) {
+          collection = figma.variables.getLocalVariableCollections().find((vr) => vr.id === theme.$figmaCollectionId);
```

Comment:
i wouldnt call getVariableCollections twice, this feels imperformant - instead do it just once? kind of like this?

also, likely we can move `figma.variables.getLocalVariableCollections()` to the top outside of the loop? like `localVariableCollections = figma.variables.getLocalVariableCollections()` and then call the find on that? that should be faster.

```suggestion
        let collection = figma.variables.getLocalVariableCollections().find((vr) => vr.id === theme.$figmaCollectionId);
        if (collection) {
```